uwblueprint/social-diversity-for-children
# Create Page For Logging Users In

### Notion ticket link



[Frontend: Create login](https://www.notion.so/uwblueprintexecs/Frontend-Create-login-838f1bd608c84748964b623f49ad72a0)



### Implementation description

- Created a login page that users will use to sign in or register to the platform



### Steps to test

1. Run the app locally and head over to http://localhost:3000/login to see the new login page

### Checklist
